### ISSUE TITLE
Reviewer 2, comment 3.3: iterative process on saturation temperature.

### ISSUE THREAD

**Issue by mostafa-razavi** [mostafa-razavi/ITIC-paper#21]
> 3.3 In Figure 5, the iterative process on vapor density is abundantly illustrated but not the iterative process on saturation temperature. This figure must be complemented by a specific graph showing how saturation temperature converges to the solution when initialized some distance away from the coexistence curve.

Response:
> The below figure is a modified version of Figure 3(b) by initializing the ITIC calculation away from coexistence curve. As can be seen, Tsat converges very fast and the sequence of (rhoV,Tsat) represented by red X symbols are practically identical even when initial Tsat estimate (black X) is deviated from true Tsat by ~10 %. We added this figure to the Supplementary Material to show how Tsat converges when it is initialized away from coexistence curve.

![image](https://user-images.githubusercontent.com/16358113/50183035-f6e97100-02de-11e9-8d26-114a0b588c65.png)

Any suggestions?

**Comment by ramess101** [comment]
@mostafa-razavi 

Please clarify what we mean by iterations, because maybe I am misunderstanding.

In this example are you saying that you perform simulations at the black X's? And that the red X's are the Tsat values computed with each iteration of solving the ITIC equations? There are not additional simulations at each red X, right?

**Comment by ramess101** [comment]
@mostafa-razavi 

OK, so not simulations but the evaluated Z and U values are obtained only at the black Xs, not the red Xs, correct?

**Comment by mostafa-razavi** [comment]
> OK, so not simulations but the evaluated Z and U values are obtained only at the black Xs, not the red Xs, correct?

Correct

**Comment by ramess101** [comment]
@mostafa-razavi 

Alright, I still think we don't want to combine this with the previous version of Figure 3. We can throw this in the SI to satisfy the reviewer though. 

However, I think a better plot would be Tsat vs iteration for each isochore just to show how quickly Tsat converges, since that is really what the reviewer wants, right?

**Comment by mostafa-razavi** [comment]
> Alright, I still think we don't want to combine this with the previous version of Figure 3. We can throw this in the SI to satisfy the reviewer though.

Yes, the above figure is now in SI.] as mentioned in the Response.

> However, I think a better plot would be Tsat vs iteration for each isochore just to show how quickly Tsat converges, since that is really what the reviewer wants, right?

If I find some time, I will do that, but I think it has a low priority. Right?

**Comment by ramess101** [comment]
@mostafa-razavi 

SI...perfect! Yes, a very low priority. From this plot it is already clear that Tsat converges within just a few iterations, it is just a matter of how tight the convergence is.
